doc/getting_started/setup_verilator.md

```diff
@@ -128,12 +167,12 @@ $ echo 'h09 l31' > gpio0-write  # Pull the pin 9 high, and pin 31 low.
 ```
 
 
-## Connect with OpenOCD to the JTAG port and use GDB
+## Connect with OpenOCD to the JTAG port and use GDB (optional)
```

Comment:
o does this mean the fpga documentation never had the gdb instructions to begin with?

Reply:
The FPGA and Verilator docs both did (bottom of https://docs.opentitan.org/doc/ug/getting_started_fpga/ and https://docs.opentitan.org/doc/ug/getting_started_verilator/) but this just makes it clear this step is "optional" I believe.